File: Windows/GameControllerSimulator/GameControllerSimulator/Bean/FeedbackReceived.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace GameControllerSimulator.Bean
{
    public class FeedbackReceived
    {
        [JsonPropertyName("large_motor")]
        public int LargeMotor { get; set; }

        [JsonPropertyName("small_motor")]
        public int SmallMotor { get; set; }

        [JsonPropertyName("led_number")]
        public int LedNumber { get; set; }
    }
}

File: iOS/GameControllerSimulator/GameControllerSimulator/Beans/FeedbackReceived.swift
//
//  FeedbackReceived.swift
//  GameControllerSimulator
//
//  Created by 猫眼螺 on 2026/8/2.
//

struct FeedbackReceived: Codable {
    
    let largeMotor: Int
    let smallMotor: Int
    let ledNumber: Int
 
    enum CodingKeys: String, CodingKey {
        case largeMotor = "large_motor"
        case smallMotor = "small_motor"
        case ledNumber = "led_number"
    }
    
}
